ParisBNF/abb/BNFabb114.xml

```diff
@@ -1297,21 +1281,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                            sitting near to the well surrounded by plants; second, <persName
                               ref="PRS6819Mary"/> giving the dog water to drink from her shoe and,
                            third, the group of virgins who chase out the dog gathered around the
-                           well. 
-                        </desc>
+                           well. </desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d35"> locus target="#87v"/> <desc> A
+                           miniture depicting Queen Walatta Qirqos (the spouse of king Ḫǝyqǝyas of
```

Comment:
I think that here, it makes sense to mark up the persons with their Bm IDs, because they are historical personalities, not protagonists.

Reply:
I see. I did for Ḫǝyqǝyas in the history element, but if I'm not mistaken, we don't have a personal name record for Walatta Qirqos (his wife) what we have is other names with the same name but with different attribution and role.

Reply:
Is this her record? https://betamasaheft.eu/persons/PRS9851Walatta/main